src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -97,21 +100,38 @@ public JMenuItem getFileMenu() {
 
     // Function to choose file in your computer
     private void importBills() {
-        System.out.println("imported");
         JFileChooser chooser = new JFileChooser();
-        //JDialog chooserDialog = chooser.createDialog(mainFrame);
         FileNameExtensionFilter filter = new FileNameExtensionFilter("Your Bills", "csv");
         chooser.setFileFilter(filter);
         int ret = chooser.showOpenDialog(mainFrame);
         if (ret == JFileChooser.APPROVE_OPTION) {
             File file = chooser.getSelectedFile();
-            loadFile(file);
+
+            // Connect with the controller to pass the file
+            List<List<String>> csvFile = convertFile(file);
+            BillImportRequestModel model = new BillImportRequestModel(csvFile);
+            this.mainFrame.getBillImportController().imports(model);
+
+            // After adding these bills, update the current bill
+            SwingUtilities.invokeLater(() -> this.mainFrame.getBillUpdateController().update(-1));
         }
     }
 
-    //Function to load file after loading file
-    public void loadFile(File file) {
-        // CONTINUE!!!
-        System.out.println("loaded");
+    // Function to convert your file after loading file
+    public List<List<String>> convertFile(File file) {
+        List<List<String>> csvFile = new ArrayList<>();
+        try {
+            BufferedReader br = new BufferedReader(new FileReader(file));
+            String line;
+            while ((line = br.readLine()) != null) {
+                String[] values = line.split(",");
+                csvFile.add(Arrays.asList(values));
+            }
+        } catch (FileNotFoundException e) {
```

Comment:
Maybe we can combine both of the exceptions together? Something like:

```suggestion
        } catch (FileNotFoundException | IOException e) {
```